app/javascript/components/internal-alert/component.jsx

```diff
@@ -30,13 +31,30 @@ const Component = ({ title, items, severity, customIcon }) => {
     [css.disableCollapse]: items?.size <= 1
   });
 
+  const renderDismissButton = handler => {
```

Comment:
Make these renderItem and renderDismissButton components in another file under components/internal-alert/components